### PR TITLE
Update bagel ml (#4)

### PR DESCRIPTION
**Pull request by ihpolash**
* update package name from betabageldb to bagelML

* update package name from betabageldb to bagelML

* update Bagel Demo Notebooks

* update Bagel Demo Notebooks

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## New Package?

Did I fill in the `tool.llamahub` section in the `pyproject.toml` and provide a detailed README.md for my new integration or package?

- [ ] Yes
- [x] No

## Version Bump?

Did I bump the version in the `pyproject.toml` file of the package I am updating? (Except for the `llama-index-core` package)

- [ ] Yes
- [x] No

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [ ] Added new notebook (that tests end-to-end)
- [ ] I stared at the code and made sure it makes sense

## Suggested Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added Google Colab support for the newly added notebooks.
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

File: docs/docs/examples/vector_stores/BagelAutoRetriever.ipynb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
```
Comment (ihpolash) [inline docs/docs/examples/vector_stores/BagelAutoRetriever.ipynb:1]: <p>Updating Bagel Package from betabageldb to bagelML. There is nothing need to be changed in the code.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/12594/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (ihpolash) [inline docs/docs/examples/vector_stores/BagelAutoRetriever.ipynb:1]: <p>Adding "BAGEL_API_KEY" to set environment. API key can be generated from <a href="https://console.bageldb.ai/dashboard" target="_blank">https://console.bageldb.ai/</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/12594/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: docs/docs/examples/vector_stores/BagelIndexDemo.ipynb [hunk 2]
```diff
@@ -1,7 +1,6 @@
 {
```
Comment (ihpolash) [inline docs/docs/examples/vector_stores/BagelIndexDemo.ipynb:1]: <p>Updating from "betabageldb" to "bagelML" to rebrand our package name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/12594/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by ihpolash** [comment]
@logan-markewich could you please review it. We are just rebranding Bagel Networks from BagelDB. Please feel free to ask if any queries.

**Review by logan-markewich — APPROVED** [review]

**Comment by logan-markewich** [comment]
LGTM!

**Pull request merged**